app/assets/html/navigation_bar.html

```diff
@@ -1,14 +1,19 @@
 <nav id="navigation" class="navbar navbar-default navbar-fixed-top">
     <div class="container">
         <div class="navbar-header">
-            <button type="button" data-toggle="collapse" data-target=".navbar-collapse" class="navbar-toggle">
+            <button ng-click="showMenu = !showMenu" type="button" class="navbar-toggle">
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
             </button>
             <a class="navbar-brand" ui-sref="home">The Colosseum</a>
         </div>
-        <div class="collapse navbar-collapse">
+        <div collapse="!showMenu" class="collapse navbar-collapse">
+            <ul class="nav navbar-nav">
+                <li>
+                    <a id="home_nav" ui-sref="home">Home</a>
```

Comment:
This was explicitly removed when the navbar was overhauled.  It seems that it was added in in d8a1b5a